src/lib/rest/rest.cpp

```diff
@@ -1128,6 +1128,21 @@ static int connectionTreat
   }
 
   //
+  // Check Content-Type and Content-Length for GET/DELETE requests
+  //
+  if ((ciP->httpHeaders.contentType != "") && (ciP->httpHeaders.contentLength == 0) && ((ciP->verb == GET) || (ciP->verb == DELETE)))
+  {
+    const char*  details = "Content-Type set but no payload supplied in GET/DELETE request";
+    OrionError   oe(SccBadRequest, details);
+
+    ciP->httpStatusCode = oe.code;
+    alarmMgr.badInput(clientIp, details);
+    restReply(ciP, oe.smartRender(ciP->version));
+    return MHD_YES;
+  }
+
+
```

Comment:
If a GET/DELETE request comes in with contentLength != 0 ... not sure what would happen.
The broker would probably complain before parsing the payload ... not sure.

What I mean with this is that this check Content-Type, Content-Len and payload could be better.

NTC (at least in this very PR)